Makefile

```diff
@@ -45,7 +50,7 @@ prod: ## Build binary for production release.
 	RUSTFLAGS="--cfg disable_faketime" cargo build ${VERBOSE} --release
 
 prod-docker:
-	RUSTFLAGS="--cfg disable_faketime --cfg docker" cargo build --verbose --release
+	RUSTFLAGS="--cfg disable_faketime --cfg docker" cargo build ${VERBOSE} --release
```

Comment:
Please revert this line. This is only used in Dockerfile.

Reply:
OK, I will revert this line then force push.